akka-stream/src/main/scala/akka/stream/javadsl/Source.scala

```diff
@@ -40,6 +40,11 @@ object Source {
    */
   def empty[O](): Source[O, NotUsed] = _empty.asInstanceOf[Source[O, NotUsed]]
 
+  /**
+    * Create a `Source` with no elements. This is an alias to `def empty[O]()`
```

Comment:
I think it should rather say `The result is the same as calling Source.<O>empty()`. (Avoid Scala syntax  in javadocs + the concept of alias means something else for Scala)

Reply:
Yes I got your point, corrected the documentation